app/views/graphiql/rails/editors/show.html.erb

```diff
@@ -55,13 +55,22 @@
 
     // Defines a GraphQL fetcher using the fetch API.
     var graphQLEndpoint = "<%= graphql_endpoint_path %>";
+    var graphQLMethod = "<%= graphql_endpoint_method %>";
     function graphQLFetcher(graphQLParams) {
-      return fetch(graphQLEndpoint, {
-        method: 'post',
+      var query = graphQLEndpoint;
+      var params = {
+        method: graphQLMethod,
         headers: { 'Content-Type': 'application/json' },
-        body: JSON.stringify(graphQLParams),
-        credentials: 'include',
-      }).then(function (response) {
+        credentials: 'include'
+      };
+
+      if (graphQLMethod.toLowerCase() === 'get') {
+        query += "?query=" + graphQLParams.query;
```

Comment:
Ohh right, `graphQLParams` is just a POJO with `query` and `variables` properties. Derp, sorry spaced on that. Should `get` requests also include query variables? 

for example

```js 
query += "&variables=" + JSON.stringify(graphQLParams.variables)
```

Or, are they being sent to the server some other way?